cpupower_gui/utils.py

```diff
@@ -97,21 +97,22 @@ def read_freqs(cpu):
 
 
 def read_freq_lims(cpu):
-    """ Reads frequency limits from sysfs """
-    try:
-        sys_path = Path(SYS_PATH.format(int(cpu)))
-        freq_minhw = int((sys_path / FREQ_MIN_HW).read_text())
-        freq_maxhw = int((sys_path / FREQ_MAX_HW).read_text())
+    """Reads frequency limits from sysfs"""
+    if is_online(cpu):
```

Comment:
I think it would be better if `freq_minhw` and `freq_maxhw` are initialised to 0 first and then return them in line 111.